ansible/roles/letsencrypt/tasks/main.yml

```diff
@@ -78,24 +78,16 @@
   - name: Force redirect to https (2/3)
     lineinfile:
       dest: /etc/apache2/sites-available/{{project_name}}.conf
-      line: "RewriteCond %{SERVER_NAME} ={{server_name}}"
-      state: present
-      insertbefore: "</VirtualHost>"
-    sudo: true
-    notify: apache restart
-
-  - name: Force redirect to https (3/3)
-    lineinfile:
-      dest: /etc/apache2/sites-available/{{project_name}}.conf
-      line: "RewriteRule ^ https://%{SERVER_NAME}%{REQUEST_URI} [END,QSA,NE,R=permanent]"
+      line: "RewriteCond %{SERVER_NAME} ={{item}}\nRewriteRule ^ https://%{SERVER_NAME}%{REQUEST_URI} [END,QSA,NE,R=permanent]"
       state: present
       insertbefore: "</VirtualHost>"
     sudo: true
+    with_items: "{{ ([server_name] + server_aliases) }}"
```

Comment:
I don't know what this does and I think it's probably not correct. did you test it?